handle guild and dm together

PandaGamersHDNL opened the pull request.

PandaGamersHDNL committed:
refactor handle guild and dm together

PandaGamersHDNL commented:
ready: when reviewed i'll remove the comments for the old system

DrunkenToast reviewed (approved):
Looks okay. Might need to be tested against all commands however if not done alrdy.

PandaGamersHDNL committed:
fix: module default true

if no corresponding module found considered as true. prevents changes in code to make it stop working, some module names are hard coded

PandaGamersHDNL committed:
refactor: remove unused import
